sgkit/tests/io/vcf/test_vcf_roundtrip.py

```diff
@@ -7,6 +7,7 @@
 from sgkit.tests.io.vcf.utils import assert_vcfs_close, path_for_test
 
 
+@pytest.mark.skip("Temporarily disabling pending bugfix; #1196")
```

Comment:
Do you think the roundtrip tests are fixable given the changes in this PR, or is there not enough information being stored in VCF Zarr to support roundtripping?

Reply:
They're definitely fixable, I was just trying to keep the scope of the PR down (plus, I'm not that familiar with the vcf writing code).

Reply:
If you think it's a straightforward fix, we could tack another commit on here for simplicity?

Reply:
I don't know how straightforward it is - the roundtrip tests are quite brittle (but valuable ones to have).

If you can get it working in this PR, that'd be great, but OK to push into another PR if you prefer.